pkg/driver/nydus/nydus.go

```diff
@@ -210,17 +211,38 @@ func (d *Driver) Convert(ctx context.Context, provider accelcontent.Provider, so
 	if err != nil {
 		return nil, errors.Wrap(err, "get source image")
 	}
-	desc, err := d.convert(ctx, provider, *image, sourceRef)
+
+	//FIXME:here we just use the first part of sourceRef as source,we need to get remote cache name gracefully
+	//split sourceRef to get remote source
+	items := strings.Split(sourceRef, "/")
+	cacheNamed := items[0] + "/library/cache-manifest:latest-nydus"
+
+	remoteCache := accelcontent.NewCache(100)
+
+	pvd, ok := provider.(*accelcontent.LocalProvider)
+	if ok {
+		remoteCache, _ = pvd.FetchRemoteCache(ctx, cacheNamed)
```

Comment:
It should give a warning message at least on error, and not fetch it if users disable the remote cache (for example no remote cache image suffix be specified).